homeassistant/components/tplink_omada/device_tracker.py

```diff
@@ -0,0 +1,106 @@
+"""Connected Wi-Fi device scanners for TP-Link Omada access points."""
+
+import logging
+
+from tplink_omada_client.clients import OmadaWirelessClient
+
+from homeassistant.components.device_tracker import ScannerEntity, SourceType
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .config_flow import CONF_SITE
+from .const import DOMAIN
+from .controller import OmadaClientsCoordinator, OmadaSiteController
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up device trackers and scanners."""
+
+    controller: OmadaSiteController = hass.data[DOMAIN][config_entry.entry_id]
+
+    clients_coordinator = controller.get_clients_coordinator()
+    site_id = config_entry.data[CONF_SITE]
+
+    # Add all known WiFi devices as potentially tracked devices. They will only be tracked if the user enables the entity.
+    async_add_entities(
+        [
+            OmadaClientScannerEntity(
+                site_id, client.mac, client.name, clients_coordinator
+            )
+            async for client in controller.omada_client.get_known_clients()
+            if isinstance(client, OmadaWirelessClient)
```

Comment:
Should we add both wireless and wired clients?

Reply:
I don't think so. Maybe if someone requests it? I think wired devices are unlikely to be portable, and reporting a turned-off computer as "Away" is a bit misleading.